truenas/webui
# NAS-121290 / 23.10 / Improvements for the apps available filter

**Testing:**

On the **Apps > Available** page:

1. Verify that the filter width looks good
2. Test that checkboxes and radiobuttons now match the design, provided [here](https://github.com/truenas/webui/discussions/7502), under **Search & Inventory** heading
3. Verify that progress bar looks properly when filter is applied
